### PR TITLE
Update anydata_type.description

### PR DESCRIPTION
**Pull request by lafernando**
Update the outdated definition of 'anydata'.

### CODE REVIEW

File: examples/anydata-type/anydata_type.bal [hunk 1]
```diff
@@ -1,30 +1,22 @@
...
-    io:println(a);
+    // In this example, the variable named `data1` of the `anydata` type holds an `int` value.
+    anydata data1 = 5;
+    io:println(data1);
 
-    // Before using the value of `a` in arithmetic operations, we need to
-    // ascertain that it is indeed an `int`. A type cast or
-    // a type guard can be used for this.
-    int intVal = <int>a;
+    // Before using the value of `data1` in arithmetic operations, we need to ascertain
```
Comment (praneesha) [inline examples/anydata-type/anydata_type.bal:8]:
```suggestion
    // Before using the value of `data1` in arithmetic operations, it is required to ascertain 
```

File: examples/anydata-type/anydata_type.bal [hunk 2]
```diff
@@ -1,30 +1,22 @@
...
+    // In this example, the variable named `data1` of the `anydata` type holds an `int` value.
+    anydata data1 = 5;
+    io:println(data1);
 
-    // Before using the value of `a` in arithmetic operations, we need to
-    // ascertain that it is indeed an `int`. A type cast or
-    // a type guard can be used for this.
-    int intVal = <int>a;
+    // Before using the value of `data1` in arithmetic operations, we need to ascertain 
+    // that it is indeed an `int`. A type cast or a type guard can be used for this.
```
Comment (praneesha) [inline examples/anydata-type/anydata_type.bal:9]:
```suggestion
    // that it is an actual `int`. A type cast or a type guard can be used for this.
```

File: examples/anydata-type/anydata_type.description [hunk 3]
```diff
@@ -1,5 +1,4 @@
-// The `anydata` type consists of pure values whose basic type is not `error`. Thus, the `anydata` type is
-// equivalent to `()|boolean|int|float|decimal|string|(anydata|error)[]|map<anydata|error>|xml|table`.
+// The `anydata` consists of plain data (a simple value, sequence value or structured value that does not contain behavioral members at any depth).
```
Comment (praneesha) [inline examples/anydata-type/anydata_type.description:1]:
```suggestion
// The `anydata` consists of plain data (a simple value, sequence value, or structured value that does not contain behavioral members at any depth). 
```

File: examples/anydata-type/anydata_type.bal (hunk 2 above)
Comment (lafernando) [reply examples/anydata-type/anydata_type.bal:9]: The meaning here should be more like "that is actually an `int`".

Comment (praneesha) [reply examples/anydata-type/anydata_type.bal:9]: ok +1

**Pull request merged**